urlchecker/core/fileproc.py

```diff
@@ -126,6 +126,9 @@ def collect_links_from_file(file_path, unique=True):
     urls = [url.strip() for url in urls if url.strip().startswith("http")]
     urls = [url.strip("\\n") if url.endswith("\\n") else url for url in urls]
 
+    # filter urls including {}
+    urls = [url for url in urls if not bool(re.search("(\\{[a-z0-9.]*})", url))]
```

Comment:
You don't need to convert to bool here, you can just do `if not re.search`

Reply:
I'll fix this tonight ;)

Reply:
@SuperKogito did you want to fix this so we can merge?

Reply:
Idk why I delayed this for tonight (facepalm). I just made the fixes, so it should be ready for merging.